sdk/data/azcosmos/CHANGELOG.md

```diff
@@ -1,9 +1,11 @@
 # Release History
 
-## 0.3.1 (Unreleased)
+## 0.4.0 (Unreleased)
```

Comment:
The convention we've been using for prerelease is to bump minor for breaking changes, and bump patch for features/bug fixes.